### DISCUSSION TITLE
Separate Backup

### DISCUSSION THREAD

**Discussion by saaiful** [spatie/laravel-backup#1526] [Ideas]
Issue: Keeping `backup` of large site with `backup:run`  created a single file with `files` & `database`. Which is ok, as we can run `backup:run --only-db` or `backup:run --only-files` to back up them separately.  The problem I face is those backup files are named as regular backup file name style like `<Y-m-d-H-i-s>.zip` and stored in same backup directory.  

I'm suggesting to add separate folders for `only-db` or `only-files` to make backups easy to manage. So when anyone run `only-db` or `only-files`  backups will be  stored in respected directory.

**Comment by erikn69** [comment]
Yo can create two disks, `files_backups`,`db_backups`
https://github.com/spatie/laravel-backup/blob/e011c04ef4aa6ba596321675d5f6a57863d25f4d/src/Commands/BackupCommand.php#L12
So
```batch
backup:run --only-files --only-to-disk=files_backups
backup:run --only-db --only-to-disk=db_backups
```
